http-gateway/cmd/http-gateway.go

```diff
@@ -54,29 +54,34 @@ func main() {
 	defer consumer.Close()
 
 	var topicExistenceChecker server.TopicExistenceChecker
+	var informerDone chan struct{}
```

Comment:
Why not create a single `done` channel and immediately defer a close operation on it. Then use that channel for both the server and informer?